src/reactviews/pages/TableDesigner/designerChangesPreviewButton.tsx

```diff
@@ -211,11 +211,20 @@ export const DesignerChangesPreviewButton = () => {
         <>
             <DialogContent className={classes.dialogContent}>
                 <ErrorCircleRegular className={classes.errorIcon} />
-                <div>{locConstants.tableDesigner.errorLoadingPreview}</div>
+                <div>
+                    {designerContext.state.generatePreviewReportResult
+                        ?.schemaValidationError ??
```

Comment:
Would be nice to have the error formatted onto multiple lines.  Are we losing newlines when it comes from STS?

Reply:
Yea, I will have to investigate more to see where this error is surfacing from. Probably will require fix in DACFX/STS. 
At least this PR corrects the behavior and matches it with ADS. 
![alt text](image.png)

Reply:
<img width="644" alt="image" src="https://github.com/user-attachments/assets/2ee5525d-9d5d-4b72-bd91-aa799bc92f87" />